### PR TITLE
docs: Document Executor from hub

### PR DESCRIPTION
**Pull request by samsja**
fixing https://github.com/jina-ai/jina/issues/4406

### CODE REVIEW

File: docs/fundamentals/executor/executor-api.md [hunk 1]
```diff
@@ -294,6 +294,29 @@ Text: hello world
 ```
 
 
+You can even pull an `Executor` from the hub and use it directly as a python object.
```
Comment (JoanFM) [inline docs/fundamentals/executor/executor-api.md:297]: Maybe we need a list of the methods from Executor API, as in the Flow. and link about Hub to give context

Comment (samsja) [reply docs/fundamentals/executor/executor-api.md]: What do you mean by a list of method ? Which part of the Flow documentation are you referring to ?

File: docs/fundamentals/executor/executor-api.md [hunk 2]
```diff
@@ -294,6 +294,30 @@ Text: hello world
 ```
 
 
+You can even pull an `Executor` from the jina hub and use it directly as a python object.
```
Comment (JohannesMessner) [inline docs/fundamentals/executor/executor-api.md:297]:
```suggestion
You can even pull an `Executor` from the Jina Hub and use it directly as a Python object.
```

File: docs/fundamentals/executor/executor-api.md [hunk 3]
```diff
@@ -294,6 +294,30 @@ Text: hello world
...
+
+
+````{admonition} More about the hub? 
+:class: information
+The [hub](https://hub.jina.ai/) is a marketplace for `Executor`.
+````
```
Comment (JohannesMessner) [inline docs/fundamentals/executor/executor-api.md:303]:
```suggestion
````{admonition} More about Jina Hub 
:class: information
The [hub](https://hub.jina.ai/) is our marketplace for `Executor`s.
````
```

File: docs/fundamentals/executor/executor-api.md (hunk 2 above)
Comment (JohannesMessner) [inline docs/fundamentals/executor/executor-api.md:297]: I have a suggestion, but I am not 100% sure it's the best; you can try it out and see.
To make the structure a bit clearer, we could use tabs for the different ways of spawning an Executor outside a Flow. That would combine the old part about Executors being a regular Python object, and your new explanation.
Something like this:

`Executor` objects can be used directly, just like any regular Python object.
There are two ways of instantiating an Executor object: From a local Python class, and from the Jina Hub.

[tab] From local Python
...

[tab] From Jina Hub
...

Btw, is there also a way to create an Executor directly form YAML? Then that could be a third tab.

File: docs/fundamentals/executor/executor-api.md [hunk 4]
```diff
@@ -292,6 +292,35 @@ print(f'Text: {da[0].text}')
...
+
+
+````{tab} From local Python
+You can pull an `Executor` from the Jina Hub and use it directly as a Python object.
+
+
+````{admonition} From Jina Hub 
+:class: information
+The [hub](https://hub.jina.ai/) is our marketplace for `Executor`s.
+````
```
Comment (JohannesMessner) [inline docs/fundamentals/executor/executor-api.md:306]: Can the info box go above both tabs? I don't think it should be inside the local python tab :)

**Review by JohannesMessner — CHANGES_REQUESTED** [review]

File: docs/fundamentals/executor/executor-api.md [hunk 5]
```diff
@@ -263,12 +263,12 @@ with f:
...
-## Other usages
+## Running Executor outside the Flow
 
-Beside running Executor inside the Flow, we list two other usages that may help you debug.  
-
-### Use Executor out of Flow
+`Executor` objects can be used directly, just like any regular Python object.
+There are two ways of instantiating an Executor object: From a local Python class, and from the Jina Hub.
 
+````{tab} From local Python
```
Comment (JohannesMessner) [inline docs/fundamentals/executor/executor-api.md:271]: The tabs don't seem to be rendered correctly: 
image
image

Also, both tabs are called 'local python'

**Review by JohannesMessner — CHANGES_REQUESTED** [review]

File: docs/fundamentals/executor/executor-api.md (hunk 5 above)
Comment (samsja) [reply docs/fundamentals/executor/executor-api.md:271]: Yes Yes I am fixing these !

**Review by JohannesMessner — APPROVED** [review]

**Review by JoanFM — APPROVED** [review]

**Pull request merged**